db/migrate/20200524080344_create_storage_systems.rb

```diff
@@ -0,0 +1,17 @@
+class CreateStorageSystems < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_systems do |t|
+      t.string :management_ip
+      t.string :name
+      t.string :secondary_ip
+      t.string :storage_array
+      t.string :storage_family
+      t.references :storage_system_type, :type => :bigint, :index => true
```

Comment:
ending a column in `_type` is going to be confusing because `_type` suffix is reserved for polymorphic relationships.  We may want to rename that table to not end in `_types`

Additionally, the storage_systems_type table is created after this point...I expect that other migration to be first.

Reply:
Could this just be `:type` and we use STI to represent the different storage types?